Makefile

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# We need all the Make variables exported as env vars.
+# Note that the ?= operator works regardless.
+
+# Enable Go modules.
+export GO111MODULE=on
+
+all: vet fmt verify test;$(info $(M)...Begin to test and verify this project.) @ ## Test and verify.
```

Comment:
I'm not necessarily opposed to having the verification and testing run by default, but the main thing this project attempts to make is a `ingress2gateway` binary, are you up for adding a `build` directive in here at the end of the chain?

Reply:
Of course, I began to consider whether to add a `build` command, but I noticed the description in READEME.md: 
```
go run .
```
So I gave up to add `build`. Since adding build is necessary, I'll add it in a later commit

Reply:
Awesome, thank you!